File: src/ts/1._arrays.ts
// Estructura de datos
/** Array */

// creación del array de autos
const AUTOS = ["Toyota", "Mazda", "McLaren", "Mercedes", "Audi"];

// acceso por posición (instantáneo)
console.log(AUTOS[2]);

// agregar al final (rápido)
AUTOS.push("Tesla");
console.log(AUTOS);

// insertar en medio (tardado)
AUTOS.splice(1, 0, "Honda");
console.log(AUTOS);

File: src/python/1._arrays.py
""" Estructura de datos
 Array """

# Creación del array de autos
autos = ["Cadillac", "Nissan", "Porsche", "Mini", "Lamborghini"]

# acceso por posicion
print(autos[2])

# agregar al final (rápido)
autos.append("Kia")
print(autos)

# insertar en medio (tardado) recorrido
autos.insert(1, "Land Rover")
print(autos)